site/content/docs/07-state-management.md

```diff
@@ -37,6 +40,10 @@ express() // or Polka, or a similar framework
 	.listen(process.env.PORT);
 ```
 
-The `session` option is a function which returns data, or a `Promise` of data, which must be serializable (using [devalue](https://github.com/Rich-Harris/devalue)), meaning that it must not contain functions or custom classes, just built-in JavaScript data types.
+`req` is an `http.ClientRequest` and `res` an `http.ServerResponse`.
+
+The session data must be serializable. This means it must not contain functions or custom classes, just built-in JavaScript data types. [devalue](https://github.com/Rich-Harris/devalue) is used for serialization.
```

Comment:
Maybe just make the reference to `devalue` a footnote. I would think most users shouldn't have to care about what's used internally

```suggestion
The session data must be serializable<sup>1</sup>. This means it must not contain functions or custom classes, just built-in JavaScript data types.
```

Then at the bottom of the page add:

1. [devalue](https://github.com/Rich-Harris/devalue) is used for serialization

Reply:
I'd suggest just removing it entirely. I agree it's an implementation detail.